src/Navigation.tsx

```diff
@@ -67,6 +67,8 @@ import {getRoutingInstrumentation} from 'lib/sentry'
 import {bskyTitle} from 'lib/strings/headings'
 import {JSX} from 'react/jsx-runtime'
 import {timeout} from 'lib/async/timeout'
+import {Welcome, WelcomeHeaderRight} from 'view/com/auth/onboarding/Welcome'
+import {RecommendedFeeds} from 'view/com/auth/onboarding/RecommendedFeeds'
```

Comment:
These should probably be in the screens folder and treated like normal screens if we're going to use the navigation model.

If we don't want to use navigation, the other option is to use the `withAuthRequired` higher-order component

Reply:
In fact, if we're using the navigation approach we need to address what happens if the user navigates away via the sidenav (currently it looks like you do so successfully but the next refresh will resume onboarding where you left off)

Reply:
Fixed.